*** pymat.cpp	Wed Jun  5 17:30:14 2002
--- pymat.cpp.ORIG	Sat Nov  4 15:30:42 2000
***************
*** 101,105 ****
      dims = mxGetDimensions(pArray);
  
!     if (nd EQ 2 AND dims[0] EQ 1 OR dims[1] EQ 1 AND dims[0] != 0 AND dims[1] != 0) {
          // It's really 1-D
          lMyDim = max(dims[0], dims[1]);
--- 101,105 ----
      dims = mxGetDimensions(pArray);
  
!     if (nd EQ 2 AND dims[0] EQ 1 OR dims[1] EQ 1) {
          // It's really 1-D
          lMyDim = max(dims[0], dims[1]);
